src/App.js

```diff
@@ -1,14 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Please your `App.js` component should be class-based and not function base.

```javascript
class App extends Component {
  render() {
    // reference as a js object
    return {
      <Calculator />
    };
  }
}
```

Reply:
Done